### PR TITLE
[RKM-28641]: Crear micro-frotend

### PR DESCRIPTION
**Pull request by manuelror**
## Type

- [ ] Bug
- [x] Task

## Related Issue

[RKM-28641](https://rankmi.myjetbrains.com/youtrack/issue/RKM-28641/22-frontend-Crear-micro-frontend)

### CODE REVIEW

File: my-app/src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import React from 'react'
 import './App.css'
+import 'antd/dist/antd.css'
+import '@rankmi/rankmi-styles/dist/css/reset-ant/main.css'
+import 'performance-results-app/dist/index.css'
 import RankmiWrapper from './rankmi-wrapper/RankmiWrapper'
-import { ExampleMainComponent } from 'rankmi-seed'
+// import { PerformanceResultsHome } from 'performance-results-app'
```
Comment (MariaMalave) [inline my-app/src/App.tsx:7]: borrar si no lo usaras

**Review by remolacho — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -27,25 +36,36 @@
     "prepare": "husky install"
   },
   "peerDependencies": {
+    "@types/enzyme": "^3.10.12",
+    "@types/enzyme-adapter-react-16": "^1.0.6",
+    "@types/react-beautiful-dnd": "^12.1.2",
```
Comment (CarlosAzocar) [inline package.json:41]: Deberian ser devdependencies

File: package.json [hunk 3]
```diff
@@ -27,25 +36,36 @@
...
     "antd": "3.26.12",
     "axios": "^0.27.2",
-    "react": "^16.0.0"
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
```
Comment (CarlosAzocar) [inline package.json:45]: Igual, devDependencies

File: package.json [hunk 4]
```diff
@@ -27,25 +36,36 @@
...
+    "react": "^16.0.0",
+    "react-beautiful-dnd": "^13.0.0"
```
Comment (CarlosAzocar) [inline package.json:47]: Igual, devdependencies

**Review by CarlosAzocar — COMMENTED** [review]
Descartar, rankmi-performance-results-app-0.1.0-beta.1.tgz

File: src/ui/alert/Alert.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    icon = getIcon(variant)
+    setIconState(icon)
+  }, [iconState])
+
+  useEffect(() => {
+    const time = setTimeout(() => {
+      alertRef.current.classList.add(styles.hide)
+      if (closeEvent) {
+        closeEvent()
+      }
```
Comment (CarlosAzocar) [inline src/ui/alert/Alert.tsx:87]: closeEvent?.()

File: src/ui/alert/Alert.tsx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    }, timeout)
+    return () => clearTimeout(time)
+  }, [timeout])
+
+  const onClose = () => {
+    timeout = 0
+    if (closeEvent) {
+      closeEvent()
+    }
```
Comment (CarlosAzocar) [inline src/ui/alert/Alert.tsx:96]: closeEvent?.()

File: src/modules/performance-results-home/PerformanceResultsHome.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  const [loading, setLoading] = useState<boolean>(false)
+  const orderedTabs = orderBy(tabs, 'position')
+
+  useEffect(() => {
+    setLoading(true)
+    getTabs()
+      .then(({ tabs }: IGeneralResponseTabs) => {
+        setTabs(serializeTabs(tabs))
+      })
+      .catch((e: any) => console.log(e))
```
Comment (CarlosAzocar) [inline src/modules/performance-results-home/PerformanceResultsHome.tsx:52]:
```suggestion
      .catch((e: any) => console.error(e))
```

File: README.md [hunk 8]
```diff
@@ -1,227 +1,181 @@
...
+```
 
-`"rankmi-seed": "link:..",`
+#### Props:
 
-<a name="src"></a>
-c) Folder `src`.
+| name      | type         | description          | required |
+|-----------|--------------|----------------------|----------|
+| `locale`  | json object  | add translation keys | true     |
```
Comment (CarlosAzocar) [inline README.md:90]: Los locale no son opcionales?

**Review by CarlosAzocar — APPROVED** [review]

**Review by marcommoyano — APPROVED** [review]

**Review by AlJor2905 — APPROVED** [review]

**Review by MariaMalave — APPROVED** [review]

**Pull request merged**